src/Schema/CMS/Events/BatchImportFlow.ts

```diff
@@ -0,0 +1,164 @@
+/**
+ * Schemas describing CMS BatchImportFlow events
+ * @packageDocumentation
+ */
+
+import { CmsContextModule } from "../Values/CmsContextModule"
+import { CmsOwnerType } from "../Values/CmsOwnerType"
+import { CmsActionType } from "./index"
+
+/**
+ * A partner has clicked the import button and started the flow.
+ *
+ * @example
+ * ```
+ * {
+ *   context_module: "batchImportFlow",
+ *   context_page_owner_type: "batchImport",
+ *   context_page_owner_id: "67b646ecbe87376bfeb3f962",
+ *   label: "click import",
+ *   referrer: ""
+ * }
+ * ```
+ */
+export interface CmsBatchImportFlowClickImport {
+  context_module: CmsContextModule.batchImportFlow
+  context_page_owner_type: CmsOwnerType.batchImport
+  context_page_owner_id: string
+  label: "click import"
+  referrer: ""
+}
+
+/**
+ * A partner has clicked to add a CSV file.
+ *
+ * @example
+ * ```
+ * {
+ *   context_module: "batchImportFlow",
+ *   context_page_owner_type: "batchImport",
+ *   context_page_owner_id: "67b646ecbe87376bfeb3f962",
+ *   label: "click add CSV"
+ * }
+ * ```
+ */
+export interface CmsBatchImportFlowClickAddCSV {
+  context_module: CmsContextModule.batchImportFlow
+  context_page_owner_type: CmsOwnerType.batchImport
+  context_page_owner_id: string
+  label: "click add CSV"
+}
+
+/**
+ * A partner has clicked to download the template.
+ *
+ * @example
+ * ```
+ * {
+ *   context_module: "batchImportFlow",
+ *   context_page_owner_type: "batchImport",
+ *   context_page_owner_id: "67b646ecbe87376bfeb3f962",
```

Comment:
Ditto here; where the partner is given the option to download we haven't yet generated an import ID. Cool if we pass an empty string here?